src/lib/containers/home.tsx

```diff
@@ -85,7 +85,10 @@ export class Home extends React.Component<Props, State> {
   }
 
   componentDidUpdate() {
-    this.listView.scrollTo({ y: this.currentScrollOffset + 1, animated: false })
+    if (this.currentScrollOffset < 0) {
```

Comment:
Yes. It ensures the PTR looks nice each time, but it does not ensure the PTR works exactly the same as it did before. I can put some more time into making it only return to 0 upon fetch if that's necessary.